README.md

```diff
@@ -1 +1,68 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
```

Comment:
- [ ] **[OPTIONAL]** To make your README file more professional, please kindly customize the [README template](https://github.com/microverseinc/readme-template) for your project. however, all headers from the template should still be present in your repo but remove or customize their contents accordingly.

Reply:
A good **README** is for others to understand what our code includes, and why it's noteworthy